### PR TITLE
One response per click for shop layers

### PR DESCRIPTION
**Pull request by loicgasser**
With this PR we make sure we only get one clipper result per selection.

### CODE REVIEW

**Comment by oterral** [comment]
lgtm

**Comment by loicgasser** [comment]
@oterral Would be also good in the tooltip to detect that we are dealing with a shop layer and retrieve only one feature to ensure we have one tooltip per layer.

**Comment by loicgasser** [comment]
I see we not using the identify service though. :smile: in the tooltip directive.

**Comment by loicgasser** [comment]
[Test Link](https://mf-geoadmin3.dev.bgdi.ch/gal_limit_ident_shop/?topic=ech&lang=fr&X=187500.00&Y=657550.00&zoom=4&bgLayer=ch.swisstopo.swissimage&layers=ch.swisstopo.digitales-hoehenmodell_25_reliefschattierung)

**Comment by oterral** [comment]
Yes you need to modify  the tooltip directive if you want to use the gaIdentify service in it.

**Comment by loicgasser** [comment]
on it

**Comment by loicgasser** [comment]
I added a commit https://github.com/geoadmin/mf-geoadmin3/pull/3347/commits/f3f9e1a57613cf5f881e7f8669bc83d571703dfb
and I am redeploying the branch. It seems to work fine.

File: src/components/tooltip/TooltipDirective.js [hunk 1]
```diff
@@ -401,29 +399,14 @@ goog.require('ga_topic_service');
...
-                  params: params
-                }).then(function(response) {
+                var tol = scope.options.tolerance;
+                var geometry = new ol.geom.Point(coordinate);
+                var returnGeometry = !!gaLayers.getLayerProperty(
+                    layerToQuery.bodId, 'highlightable');
+                var limit = gaLayers.getLayerProperty(
+                    layerToQuery.bodId, 'shop') ? 1 : null;
+                all.push(gaIdentify.get(map, [layerToQuery], geometry, tol,
+                    returnGeometry, null, limit).then(function(response) {
```
Comment (oterral) [inline src/components/tooltip/TooltipDirective.js]: instead of null you should put `canceler.promise` as timeout.

**Comment by oterral** [comment]
just a comment about the timeout param otherwise lgtm

**Comment by loicgasser** [comment]
addressed thanks for the review!

**Comment by oterral** [comment]
Nice thx for taking care of this.

**Comment by loicgasser** [comment]
ping @pauloamado for next deploy

**Comment by gjn** [comment]
Very nice.

Please squash and merge.

**Pull request merged**